src/_assets/css/main.scss

```diff
@@ -1379,5 +1379,13 @@ blockquote {
   padding: 10px 20px 0 20px;
 }
 
+iframe[src*="dartpad.dev"] {
+  border: 1px solid #ccc;
+  margin-bottom: 1rem;
+  min-height: 150px;
```

Comment:
I think this might be a little too small. 250px seems to be a better height for most snippets.

When I try the null safety codelab, I lose visibility of the console output when I uncomment the line:

<img width="995" alt="Screen Shot 2020-06-09 at 8 26 08 PM" src="https://user-images.githubusercontent.com/1145719/84223708-79479100-aa8f-11ea-817f-fe15bc597d4d.png">